stable/anchore-admission-controller/templates/deployment.yaml

```diff
@@ -3,6 +3,9 @@ kind: Deployment
 metadata:
   name: {{ template "anchore-admission-controller.fullname" . }}
   labels: {{- include "anchore-admission-controller.labels" . | nindent 4 }}
+  {{- with .Values.labels}}
```

Comment:
We should include these `with` statements in the `anchore-admission-controller.labels` helper template rather than sticking them into each resource template directly. This will help keep our templates DRY.